client/src/app/shared/shared-share-modal/video-share.component.ts

```diff
@@ -204,15 +206,19 @@ export class VideoShareComponent {
     if (this.playlist) {
       this.playlistUrl = await this.getPlaylistUrl()
       this.playlistEmbedUrl = await this.getPlaylistEmbedUrl()
-      this.playlistEmbedHTML = await this.getPlaylistEmbedCode()
-      this.playlistEmbedSafeHTML = this.sanitizer.bypassSecurityTrustHtml(this.playlistEmbedHTML)
+      this.playlistEmbedHTML = await this.getPlaylistEmbedCode(this.customizations.responsive)
+      this.playlistEmbedSafeHTML = this.sanitizer.bypassSecurityTrustHtml(this.customizations.responsive
+        ? await this.getPlaylistEmbedCode()
+        : this.playlistEmbedHTML)
     }
 
     if (this.video) {
       this.videoUrl = await this.getVideoUrl()
       this.videoEmbedUrl = await this.getVideoEmbedUrl()
-      this.videoEmbedHTML = await this.getVideoIframeCode()
-      this.videoEmbedSafeHTML = this.sanitizer.bypassSecurityTrustHtml(this.videoEmbedHTML)
+      this.videoEmbedHTML = await this.getVideoEmbedCode(this.customizations.responsive)
+      this.videoEmbedSafeHTML = this.sanitizer.bypassSecurityTrustHtml(this.customizations.responsive
```

Comment:
I think `this.sanitizer.bypassSecurityTrustHtml(await this.getVideoEmbedCode({ responsive: false }))` would be simpler

Reply:
Done in https://github.com/Chocobozzz/PeerTube/pull/5690/commits/c26c1a5f735bd6ca1d4957072a741b1f8c953e23